srcpkgs/meson/patches/dont-use-native-pkgocnfig-for-gir.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/mesonbuild/modules/gnome.py b/mesonbuild/modules/gnome.py
+index bf49770..42473cb 100644
+--- mesonbuild/modules/gnome.py
++++ mesonbuild/modules/gnome.py
+@@ -429,7 +429,7 @@ class GnomeModule(ExtensionModule):
+         try:
+             gir_dep = self.gir_dep or PkgConfigDependency('gobject-introspection-1.0',
+                                                           state.environment,
+-                                                          {'native': True})
++                                                          {'native': False})
+             pkgargs = gir_dep.get_compile_args()
+         except Exception:
+             raise MesonException('gobject-introspection dependency was not found, gir cannot be generated.')
+
```

Comment:
This means pkg-config is called through qemu? Is there a way to adapt our wrapper instead?